### PR TITLE
Fixed issue where space character wouldn't appear in last update time of a library

### PR DESCRIPTION
**Pull request by thecosmicfrog**
e.g. "15minutes ago" instead of "15 minutes ago".

This was due to a literal space (' ') being used in the "_ago" string rather than the special "&#160;" character.

### CODE REVIEW

**Comment by Logan676** [comment]
@thecosmicfrog I have noticed that there still be another [4 strings](https://github.com/haiwen/seadroid/blob/master/res/values/strings.xml#L280) not being corrected yet. Please change all of them in order this PR to be merged. Thanks.

**Comment by thecosmicfrog** [comment]
Hi Logan. There was nothing wrong with line 280 from what I can see. I verified this by looking at the running app. It's only preceding and trailing spaces that seem to cause issue.

Regardless, I have committed some general formatting and grammatical fixes, including fixing some spelling mistakes. Let me know what you think.

Thanks,
Aaron

**Comment by Logan676** [comment]
Though there was nothing wrong with line 280 from what you have seen when running the app, we\`d better keep the string format being consistent. I have noticed the [official Android app](https://github.com/google/iosched/blob/master/android/src/main/res/values/strings.xml#L641) even didn\`t use the special character `&#160;` through the whole file, so we\`d better remove them all as well.

I know there are still some other strings not being consistent in the file, like `...` should be changed to  `&#8230;`, use `plural tags` to handle the plural. At least for an elegant and cleaner open sourced project, that\`s what we are working for. :smile: 

Regards
Logan

**Comment by thecosmicfrog** [comment]
The problem I noticed was that there would be no space at all in "minutes ago", "second ago", etc. This is a bad UI issue. Using the special character should solve this, but no point in using them where they are not needed.

What are you proposing?

**Comment by Logan676** [comment]
My meaning is
1. agree with you, use `Seafile` instead of `seafile`
2. remove all `&#160;`, use a literal space (' ') instead.

**Comment by thecosmicfrog** [comment]
Ah, okay cool. However, I believe the issue with "5minutes remaining", "6hours remaining" (i.e. no space between the digit and the words) will persist without the &#160; character.

What else would be causing this?

**Comment by Logan676** [comment]
we can change it like this ` 5 minutes remaining`, ` 6 hours remaining` or `some words `, 
instead of `&#160;5 minutes`, `&#160;6 hours remaining` or `some words&#160;`

**Comment by thecosmicfrog** [comment]
But the original string read as follows:
`<string name="minutes_ago"> minutes ago</string>`

...note the singular space before "minutes", or more verbosely:

`<string name="minutes_ago">SPACEminutes ago</string>`

This did not work as it was, so I corrected it by removing the singular space altogether and replacing it with a `&#160;` as such:

`<string name="minutes_ago">&#160;minutes ago</string>`

File: res/values/strings.xml [hunk 1]
```diff
@@ -209,22 +209,22 @@
...
     <string name="lockpattern_settings_help_how_to_record">Learn how to draw Gesture Lock Pattern</string>
-    <string name="lockpattern_recording_incorrect_too_short">Link at least 4 points, try again!</string>
+    <string name="lockpattern_recording_incorrect_too_short">Swipe across at least 4 points, try again!</string>
     <string name="lockpattern_pattern_entered_header">Pattern saved</string>
     <string name="lockpattern_need_to_draw">Please draw Gesture Lock pattern</string>
     <string name="lockpattern_need_to_confirm">Please confirm your pattern</string>
-    <string name="lockpattern_need_to_unlock_wrong">Pattern not match, try again!</string>
-    <string name="lockpattern_pattern_confirmed_header">Confirm to save your pattern</string>
-    <string name="lockpattern_recording_inprogress">Move away your firgure after drawing"</string>
+    <string name="lockpattern_need_to_unlock_wrong">Pattern didn't match. Try again.</string>
```
Comment (Logan676) [inline res/values/strings.xml]: I pulled your commit and run the project, the IDE complained an error, here "didn\`t" should be changed like this "didn\\`t"

**Comment by thecosmicfrog** [comment]
Argh, that was a silly oversight. Apologies. Fix committed.

**Comment by Logan676** [comment]
@thecosmicfrog please rebase on the original master.

**Comment by thecosmicfrog** [comment]
@Logan676 Can you tell me exactly what I should run?

**Comment by Logan676** [comment]
1. git fetch origin
2. git rebase origin/master
3. resolve conflicts if any
4. git push -f yourbranch

please Google it for details.

**Comment by thecosmicfrog** [comment]
Ah sorry, I misunderstood. That's fine. I think that should be alright now?

**Pull request closed without merging**